Documentation/operations/performance/scalability/identity-relevant-labels.rst

```diff
@@ -120,21 +120,23 @@ and the default inclusive labels will be used to evaluate Cilium identities:
 
 .. code-block:: yaml
 
-    labels: "k8s:io.kubernetes\\.pod\\.namespace k8s:k8s-app k8s:app k8s:name"
+    labels: "io\\.kubernetes\\.pod\\.namespace k8s-app app name kind$ other$"
 
 The above configuration would only include the following label keys when
 evaluating Cilium identities:
 
-- k8s:k8s-app
-- k8s:app
-- k8s:name
+- k8s-app
+- app
+- name
```

Comment:
```suggestion
- ``.*k8s-app.*``
- ``.*app.*``
- ``.*name.*``
```

Reply:
explained in the above

Reply:
this should be `k8s-app.*` etc based on my understanding